renderdoc/driver/gl/wrappers/gl_draw_funcs.cpp

```diff
@@ -709,10 +709,47 @@ bool WrappedOpenGL::Serialise_glDrawArrays(GLenum mode, GLint first, GLsizei cou
   return true;
 }
 
+void WrappedOpenGL::CopyClientMemoryArrays(GLint first, GLsizei count)
+{
```

Comment:
Can you have a `m_State < WRITING || VAO_is_bound` early out here? That way we avoid some per-drawcall overhead of running through the loop doing queries only to find out it's not fake VBOs bound.

In fact, do we want to do this at all except for in `WRITING_CAPFRAME` ? otherwise we're memcpy'ing around a lot of data that we don't care about. Since we're late-fetching the data at draw time, I don't think we need this for anything during idle.